core/sail/base/src/main/java/org/eclipse/rdf4j/sail/base/Changeset.java

```diff
@@ -348,7 +377,28 @@ protected void setChangeset(Changeset from) {
 		this.statementCleared = from.statementCleared;
 	}
 
+	public static Changeset simpleClone(Changeset from) {
```

Comment:
Couldn't this be an instance method as easily? And why `simpleClone` rather than `clone`? Either way, might be good to stick a small javadoc on this to explain its purpose.

Reply:
Maybe it would be better to call it a shallow clone because it doesn't clone any of the data structures and instead just copies of the the references (pointers). 

I'm not sure why I went with a static method, I'll try to change is to an instance method and see what happens.